src/mediarithmics/plugins/common/BasePlugin.ts

```diff
@@ -0,0 +1,331 @@
+import * as express from "express";
```

Comment:
Question: As Git has lost the connection between this file and the original `BasePlugin` file, won't this break everything when do the merge with master?

Reply:
(well, I didn't see all the conflicts...)

Reply:
I think I solved the conflict when rebasing

Reply:
Okay! I'll test it